src/Repository/WrongAggregate.php

```diff
@@ -4,11 +4,9 @@
 
 namespace Patchlevel\EventSourcing\Repository;
 
-use RuntimeException;
-
 use function sprintf;
 
-final class WrongAggregateException extends RuntimeException
+final class WrongAggregate extends RepositoryException
 {
     public function __construct(string $aggregateClass, string $expected)
```

Comment:
We should update the message here too, cannot add a suggestion since the line wasnt change:
`Wrong aggregate given: got '%s' but expected '%s'`